src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs

```diff
@@ -38,9 +38,20 @@ public sealed class DownloadFromAzure : AzureConnectionStringBuildTask
 
         public int MaxClients { get; set; } = 8;
 
+        [Output]
+        public ITaskItem[] PathTooLongBlobs
+        {
+            get
+            {
+                return pathTooLongEntries.ToArray();
```

Comment:
\<nit> Expression-bodied member syntax?

Reply:
Ooh, Didn't know about that one. Looks much better!